### ISSUE TITLE
Viewer Configuration Error When No Background Map is Defined

### ISSUE THREAD

**Issue by fjlopez** [sitmun/sitmun-viewer-app#22]
**Describe the bug**
When a background map is not defined, the viewer displays a configuration error (`baseLayers` configuration is incorrect) and does not load the rest of the settings.

**To Reproduce**
1. Select an application without a background map
2. See error

**Expected behavior**
The viewer should not require having a background map as mandatory.

**Screenshots**
<img width="454" alt="image" src="https://github.com/sitmun/sitmun-viewer-app/assets/3294927/db7c951f-69e0-45f0-86ac-4992a92f5730">

**Comment by codinachssm** [comment]
In fact, the error message about "baseLayer" changed to "layerCatalog". I don't know if after fixing the problem of "layerCatalog", the error message of "baseLayer" will disappear or not.

**Comment by fjlopez** [comment]
I think that these are two different problems. `baseLayer` happens when no background is defined, `layerCatalog` happens when a background is defined but fails to be interpreted.  This is #27

**Comment by codinachssm** [comment]
Yes, you are right. But even having a defined background map this error was occurring. But suddenly #27 appeared. I don't know if when #27 is fixed it will appear again or not.
